libraries/ArduinoOTA/ArduinoOTA.h

```diff
@@ -93,6 +93,7 @@ class ArduinoOTAClass
     void _onRx(void);
     int parseInt(void);
     String readStringUntil(char end);
+	bool constantTimeEquals(const String &string1, const String &string2);
```

Comment:
Suggestion: can this method be moved into String class, so reuse in other places is possible?

Reply:
Ok I'll do that